### DISCUSSION TITLE
Thermal Loads

### DISCUSSION THREAD

**Discussion by Corentin1985** [JWock82/Pynite#110] [Ideas]
Hi, is it possible to add thermal loads? 
For instance, assuming ambient temperature is 20°C, and assuming the beams are at X°C (for instance 200°C ou -10°C), what would be the efforts on the fixed nodes?

**Comment by JWock82** [comment]
`PyNite` doesn't do thermal loads yet. I don't think they'd be too difficult to implement, but I just haven't gotten around to it yet. My focus has been on further developing the plate elements lately.

I'm only 1 person doing this in my free time, so I usually work on the features that are most useful to me. Recently I added a discussion called "Priorities for Future Development". I've been using it as a "suggestion box" and I plan to start working on the hot items suggested there once I get enough votes in.

Of course, if anyone wants to financially sponsor a feature I'd be happy to bump that feature to the top of the "to-do" list.

**Reply by Corentin1985** [reply]
Ok, thanks. How much should that support be to be on top?

**Reply by JWock82** [reply]
We’ll.. to be honest… you’re the first person to ask me that. I guess I don’t know. What’s it worth to you?

**Reply by Corentin1985** [reply]
Well, I don't really know either. That would not be for me, but for a company. I'll keep this discussion in mind if the need becomes more pressing. Best regards.

**Reply by JWock82** [reply]
I think it's a matter of making sure it's a win-win for everyone. It's got to be worth it to your company, and worth it to me. I think thermal loading would be fairly straightforward to implement. I've got several texts that show explicitly how to do it. Sometimes I get into simple things though and find they are not so simple.
